### PR TITLE
Download the latest Syncthing binaries when compiling on windows

### PR DESCRIPTION
**Pull request by Zillode**
Could be improved by rewriting `build-syncthing.sh` in groovy such that it builds on linux and windows.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -111,6 +117,35 @@ def getVersionCodeFromManifest() {
     return Integer.parseInt(matcher.group(1))
 }
 
+task downloadNative(type: Copy) {
+    def release = new JsonSlurper().parseText("https://api.github.com/repos/syncthing/syncthing/releases/latest".toURL().text)
+
+    // Download ARM
+    new File('bin').mkdirs()
+    copy {
+        ant.get(src: "https://github.com/syncthing/syncthing/releases/download/v0.10.30/syncthing-linux-arm-"+release.tag_name+".tar.gz", dest: "bin/syncthing-arm.tar.gz")
```
Comment (Nutomic) [inline build.gradle]: This doesn't make sense, you're hardcoding the version in one place and reading it from the API in another?

**Comment by Nutomic** [comment]
Is there a way to read the version from the git submodule? Anything else will break if the syncthing API changes.

**Comment by Zillode** [comment]
- I don't think this is a big issue because we're always up to date.
- It would need a `git submodule init` to fetch the tagged Syncthing version, which means relying on git, which could also not be installed.

But sure, closed.

**Comment by Nutomic** [comment]
I'm not saying we can't do this, it's just that I don't really like it.

Looks like we don't have any other option, so we'll have to do this. But we should have some kind of disclaimer that windows builds might randomly break.

**Comment by Nutomic** [comment]
@Zillode As I said, I would merge this if you include a disclaimer in the readme.

**Comment by Nutomic** [comment]
What's the status on this, can we merge it? As I said, all I want is a note in the readme that builds might break ;)

**Comment by Nutomic** [comment]
Closing because this is outdated (feel free to reopen).

**Pull request closed without merging**